packages/mcp-server/src/tools.ts

```diff
@@ -457,6 +461,133 @@ export const TOOL_HANDLERS = {
       apiService,
     });
   },
+  get_event_attachment: async (context, params) => {
+    const apiService = apiServiceFromContext(context, {
+      regionUrl: params.regionUrl,
+    });
+
+    if (!params.organizationSlug) {
```

Comment:
shouldn't need any validation for the params that aren't marked as optional - the function signature is a bit weird if you call it directly, but the tool service itself should be validating them just fine